trueos-handbook/helpsupport.rst

```diff
@@ -455,6 +455,77 @@ If sound issues persist, consider asking the :ref:`TrueOS Community` for
 help or :ref:`Report a Bug`. When reporting an issue, be sure to include
 both the version of |trueos| and name of the sound card.
 
+.. index:: upgrade from PCBSD
+.. _Upgrading from PCBSD 10.x to TrueOS:
+
+Upgrading from |pcbsd| 10.x to |trueos|
+=======================================
+
+.. warning:: If any user account uses PersonaCrypt, please be sure to
+   save any encryption keys to a safe place (e.g. a thumb drive) before
+   beginning the upgrade process. Loss of encryption keys may result in
+   being unable to import the home directory after the upgrade is
+   complete.
+
+If the system is using |pcbsd| 10.x, the option to update to |trueos|
+does not appear in the Control Panel version of Update Manager. This is
+because a new installation is required in order to migrate to |trueos|.
+However, the |trueos| installer allows the user to keep all their
+existing data and home directories as it provides the ability to
+install |trueos| into a new boot environment. In other words, the new
+operating system and updated applications are installed while the ZFS
+pool and any existing boot environments are preserved. Since the new
+install is in a boot environment, the option to boot back into the
+previous |pcbsd| installation remains.
+
+.. note:: This option overwrites the contents of :file:`/etc`. If any
+   custom configurations exist, save them to a backup or the home
+   directory first. Alternately, use the |sysadm|
+   :sysclbk:`Boot Environment Manager <boot-environment-manager>`
+   post-installation to mount the previous |pcbsd| boot environment to
+   copy over any configuration files which may not have been backed up.
+
+To perform the installation to a new boot environment, start the
+|trueos| installation as described earlier in the chapter. In the
+:ref:`System Selection` screen, choose to install either a desktop or
+a server. Press :guilabel:`Next` to view the :guilabel:`Disk Selection`
+screen, shown in :numref:`Figure %s <upgrade1>`.
+
+.. TODO update screen when text reworks are added to STABLE.
```

Comment:
Probably should double check this screen and remove the TODO if the screenshot has already been updated.